lightningd/peer_control.h

```diff
@@ -133,6 +133,11 @@ static inline bool peer_on_chain(const struct peer *peer)
 	return peer_state_on_chain(peer->state);
 }
 
+static inline bool peer_has_error(const struct peer *peer)
+{
+  return peer->error == NULL || peer->error[0] == '\0';
```

Comment:
should has error be checking if peer->error is equal to NULL or not equal to NULL?

Reply:
you're right! brain malfunction :|